packages/desktop/views/account/multi-step/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as ShowOneTimeAddress } from './ShowOneTimeAddress.svelte'
+export { default as GenerateMana } from './GenerateMana.svelte'
+export { default as SignTransaction } from './SignTransaction.svelte'
+export { default as MultiStep } from './MultiStep.svelte'
```

Comment:
Sorry for being annoying with the names, but I think the names of the `steps` are more accurate than these 🥲 
`ShowOneTimeAddress` -> `OneTimeDeposit`
`GenerateMana` -> `FundConfirmation`
`SignTransaction` -> `AccountCreation`
wdyt?